src/app/owner/owner.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="row">
+    <div class="col-md-5">
+        <app-owner-list></app-owner-list>
+    </div>
+    <div class="col-md-7">
+        <router-outlet></router-outlet>
```

Comment:
<router-outlet></router-outlet> should only be done once in the app component thats it